lib/common/socket/evloop.c.h

```diff
@@ -624,6 +624,9 @@ int h2o_evloop_run(h2o_evloop_t *loop, int32_t max_wait)
     if (h2o_sliding_counter_is_running(&loop->exec_time_counter)) {
         update_now(loop);
         h2o_sliding_counter_stop(&loop->exec_time_counter, loop->_now_millisec);
+
+        if (h2o_sliding_counter_is_running(&loop->exec_time_nanosec_counter))
```

Comment:
Do we need this `if`? It is my understanding that `exec_time_nanosec_counter` and `exec_time_counter` are used the same way. Assuming that is the case, I think that the `if` is superfluous.

Reply:
Agreed. This conditional is not needed, especially if we're replacing `exec_time_nanosec_counter` with `exec_time_nanosec_counter`, which I have done in: 9f01afea